torchvision/io/video_reader.py

```diff
@@ -119,14 +146,29 @@ def __next__(self) -> Dict[str, Any]:
             and corresponding timestamp (``pts``) in seconds
 
         """
-        if self.is_cuda:
+        if self.backend == "cuda":
             frame = self._c.next()
             if frame.numel() == 0:
                 raise StopIteration
             return {"data": frame}
```

Comment:
I wonder if we should just return None as pts until we fix the cuda side, don't have strong opinion but seems more consistent to return a dictionary with the same keys for all backends

Reply:
Probably a good idea, yeah.